website/docs/01 - Introduction/03 - Setup and Configuration.md

```diff
@@ -10,352 +10,7 @@ In order to communicate to the MultiValue database, the following are required:
-
-    * compiling with override and case-insensitive flags
-    udtexecute 'basic ':sourceDir:' ':programName:' -d -o -i' returning errmsg
-    if @system.return.code then
-      call error_handler(ERROR_DEPLOYMENT, output)
-      go response
-    end
-
-    * cataloging with default catalog (global) with force override
-    udtexecute 'catalog ':sourceDir:' ':programName:' force' returning errmsg
-    if @system.return.code then
-      call error_handler(ERROR_DEPLOYMENT, output)
-      go response
-    end
-
-    if udoSetProperty(output, 'deployed', programName) then
-      call error_handler(ERROR_UDO, output)
-      go response
-    end
-  case action eq 'subroutine'
-    * call specified subroutine
-    if udoGetProperty(input, 'subroutineId', subroutineId, type) then
-      call error_handler(ERROR_MALFORMED_INPUT, output)
-      go response
-    end
-
-    if udoGetProperty(input, 'options', options, type) then
-      call error_handler(ERROR_MALFORMED_INPUT, output)
-      go response
-    end
-
-    if udoGetProperty(input, 'setupId', setupId, type) then
-      call error_handler(ERROR_MALFORMED_INPUT, output)
-      go response
-    end
-
-    if udoGetProperty(input, 'setupOptions', setupOptions, type) then
-      call error_handler(ERROR_MALFORMED_INPUT, output)
-      go response
-    end
-
-    if udoGetProperty(input, 'teardownId', teardownId, type) then
-      call error_handler(ERROR_MALFORMED_INPUT, output)
-      go response
-    end
-
-    if udoGetProperty(input, 'teardownOptions', teardownOptions, type) then
-      call error_handler(ERROR_MALFORMED_INPUT, output)
-      go response
-    end
-
-    call @setupId(setupOptions)
-
-    call @subroutineId(options, output)
-
-    call @teardownId(teardownOptions)
-  case 1
-    call error_handler(ERROR_UNSUPPORTED_ACTION, output)
-    go response
-end case
-
-response:
-  * if an error occurs in serializing the output object it will likely result in a fatal error
-  * there is little else we can do here except let that fatal error occur
-  x = udoWrite(output, UDOFORMAT_JSON, outputSerialized)
-
-  * if an error occurs in freeing the memory space for the objects, we will ignore it as there is little else that can be done
-  x = udoFree(output)
-  x = udoFree(input)
-
-  return; * returning to caller
-
-subroutine error_handler(errorCode, output)
-
-* note that this routine intentionally does not react to udo errors because it could
-* potentially result in an unending sequence of errors
-
-x = udoSetProperty(output, 'errorCode', errorCode)
-
-return
-```
-
-Upon clicking "OK", MVIS Admin will allow for the source code to be deployed.
+MVOM will automatically create a REST subroutine definition and deploy the subroutine to the database server if not present. The created subroutine definition is named `mvom_main` and is suffixed by an 8 character hash of the bundled source code. A new version of MVOM will result in a new hash and associated subroutine deployment.
```

Comment:
"..._will_ result..." is not necessarily accurate.  Only if the bundle changes will a new subroutine hash be generated.  "_may_ result"  might be a better phrasing here.